routes/membership.js

```diff
@@ -0,0 +1,123 @@
+
+stripe = require("stripe")("sk_test_BQokikJOvBiI2HlWgH4olfQ2");
+module.exports = function(app) {
+
+    app.get('/membership/create', function(req, res, next) {
+        console.log("test");
+        res.render('payment/paywall.jsx');
+    });
+
+
+
+//single token that represents the info the customer has
```

Comment:
what is this object doing? it's not being assigned to a variable or anything. maybe just extra copy/pasted stuff from stripe? if so, can delete?